client/client.go

```diff
@@ -260,8 +260,14 @@ func (c httpMetricClient) retryWithBackoff(
 			log.Errorf("error encountered while retrieving upload location: %v", err)
 			continue
 		}
+		log.Infof("Successfully retrieved upload URL with retry=%v", i)
 
 		resp, err = c.buildAndDoRequest(metricFile, uploadURL, agentVersion, UID, hash)
+		if err != nil {
+			log.Infof("Successfully put sample with retry=%v", i)
```

Comment:
We should probably have the logs set at the DEBUG level so not all healthy clusters get spammed with these logs. I also noticed a 'verbose' flag that could be used instead of adding new logs.